### PR TITLE
docs: replaced merico-dev/lake by apache/incubator-devlake

### PR DESCRIPTION
**Pull request by billmetangmo**
# Summary

### Key Points

- [ ] This is a breaking change
- [X ] New or existing documentation is updated

### Description
Change the url that link from the previous repo (merico-dev) to the new one ( apache)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ DevLake is designed for developer teams looking to make better sense of their de
 
 #### Launch DevLake
 
-1. Download `docker-compose.yml` and `env.example` from [latest release page](https://github.com/merico-dev/lake/releases/latest) into a folder.
+1. Download `docker-compose.yml` and `env.example` from [latest release page](https://github.com/https://github.com/apache/incubator-devlake/wiki/How-to-Set-Up-Dev-Lake-with-Tin/releases/latest) into a folder.
```
Comment (klesh) [inline README.md:80]: this doesn't seem right.

**Review by klesh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (billmetangmo) [reply README.md]: done

**Review by klesh — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -5,8 +5,8 @@
 # DevLake
 
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat&logo=github&color=2370ff&labelColor=454545)](http://makeapullrequest.com)
-![badge](https://github.com/merico-dev/lake/actions/workflows/test.yml/badge.svg)
-[![Go Report Card](https://goreportcard.com/badge/github.com/merico-dev/lake)](https://goreportcard.com/report/github.com/merico-dev/lake)
+![badge](https://github.com/https://github.com/apache/incubator-devlake/wiki/DataModel.Domain-layer-schema/actions/workflows/test.yml/badge.svg)
```
[the remark this answers is not in the document]
Comment (klesh) [reply README.md:8]: No, I dont think it was fixed. I'm still seeing 2 `https` in one url.
image

**Review by klesh — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (billmetangmo) [reply README.md]: Yes, i was thinking , I have done it.
I also do the update from merico-dev/lake to apache in chinese README

**Comment by klesh** [comment]
Hi, @billmetangmo . Seems like some changes of this PR had been submitted by other PRs during review process.
Would you mind to rework this PR,  verify if any missing pieces you would like to resubmit?

**Comment by warren830** [comment]
> # Summary
> ### Key Points
> * [ ]  This is a breaking change
> * [X ] New or existing documentation is updated
> 
> ### Description
> Change the url that link from the previous repo (merico-dev) to the new one ( apache)

Hi Bill, may I have your email address? We'd like invite you to join our slack on https://join.slack.com/t/devlake-io/shared_invite/zt-18uayb6ut-cHOjiYcBwERQ8VVPZ9cQQw

**Comment by billmetangmo** [comment]
> Hi, @billmetangmo . Seems like some changes of this PR had been submitted by other PRs during review process. Would you mind to rework this PR, verify if any missing pieces you would like to resubmit?

Nothing to add , everything is corrected. I will close the PR.

**Comment by billmetangmo** [comment]
> > # Summary
> > ### Key Points
> > 
> > * [ ]   This is a breaking change
> > * [X ] New or existing documentation is updated
> > 
> > ### Description
> > Change the url that link from the previous repo (merico-dev) to the new one ( apache)
> 
> Hi Bill, may I have your email address? We'd like invite you to join our slack on https://join.slack.com/t/devlake-io/shared_invite/zt-18uayb6ut-cHOjiYcBwERQ8VVPZ9cQQw

Yes. I saved it on this link https://www.3hg.fr/CHATONS/Pastebin/?bbbf2ffb40acab35#FJEXFAFxzgATDYN1bptZLSn4kaiAxPTuh6baWw7YA2tL  for privacy. Be sure to save it somewhere before closing your browser tab because it will be destroyed after the first read.

**Pull request closed without merging**